contexts/social-context.provider.tsx

```
@@ -23,19 +22,20 @@ const SocialContext = createContext<SocialContextProps>({
   socials: [],
   primarySocial: null,
   isFetchingSocials: false,
-  setPrimarySocial: (social: ISocials) => {},
+  setPrimarySocial: (social: ISocials) => { },
 });
 
 export const SocialContextProvider = ({ children }: PropsWithChildren<{}>) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)